mqttwarn/services/pushsafer.py

```diff
@@ -101,7 +104,16 @@ def plugin(srv, item):
         params['ex'] = addrs[10]
 
     if len(addrs) > 11:
-        params['a'] = addrs[11]
+        params['cr'] = addrs[11]
```

Comment:
Reassigning slot 11 to another parameter will break backwards-compatibility.

Reply:
i changed the sorting

Reply:
Thanks! I think your patch will be good to go. I will readjust it correspondingly after bringing in GH-629.